src/builders/enzymes.jl

```diff
@@ -297,7 +297,7 @@ function simplified_enzyme_constraints(;
                 contribution(fluxes_reverse, mass_cost_reverse, f) for f in fs;
                 init = zero(C.LinearValue), # TODO not type stable if LinearValueT{not Float64}
             ),
-            bound,
+            C.BetweenT(bound...),
```

Comment:
this is a keyword argument (notice the `;` before), needs a name here.

Also you have to make sure that the possibilities of what the bound is are handled well (this will cease working if someone supplies an equality bound)

Reply:
changing like you suggest in the next comment, but the default constructor should still be available right? I.e. `BetweenT(a,b)` should just make the bound, just like `BetweenT(;lower=a, upper=b)`

Reply:
no, if you type BetweenT in the code you need to bump the dependency.

I'd instead very strongly recommend not adding additional magicks (this depends on yet another undeclared dependency that the type used has to support `zero()`! maybe document it fully?) and support only the old behavior with Between, together with supporting the generic behavior where people actually add the capacity directly via bound `isa Bound` -- which you then use in the other software